### ISSUE TITLE
Wrong season and wrong episode detected (due to x264)

### ISSUE THREAD

**Issue by ratoaq2** [guessit-io/guessit#295]
```
+-------------------------------------------------------+
+                   GuessIt 2.0.5                       +
+-------------------------------------------------------+
|      Please report any bug or feature request at      |
|     https://github.com/guessit-io/guessit/issues.     |
+-------------------------------------------------------+
```

Given any show with the standard SxxExx match, when there's x264 (or x265) match just after the episode match, then guessit gets confused:
```
guessit -t episode Some.Show.S02E14.X264.1080p.HDTV
For: Some.Show.S02E14.X264.1080p.HDTV
GuessIt found: {
    "title": "Some Show", 
    "season": [
        2, 
        14
    ], 
    "video_codec": "h264", 
    "screen_size": "1080p", 
    "format": "HDTV", 
    "type": "episode"
}
```

**Comment by ratoaq2** [comment]
There's this strange case as well:

`guessit 2.0.5`

```
guessit -t episode "Game.of.Thrones.S6.Ep5.X265.Dolby.2.0.KTM3.mp4"
For: Game.of.Thrones.S6.Ep5.X265.Dolby.2.0.KTM3.mp4
GuessIt found: {
    "title": "Game of Thrones", 
    "season": [
        6, 
        5
    ], 
    "video_codec": "h265", 
    "episode_title": "Dolby", 
    "audio_channels": "2.0", 
    "release_group": "KTM3", 
    "container": "mp4", 
    "mimetype": "video/mp4", 
    "type": "episode"
}
```

**Comment by labrys** [comment]
This appears to be related to #286

**Comment by Toilal** [comment]
I also think it's related. I still didn't spend more time on it.

**Comment by ratoaq2** [comment]
The second case fails to detect DolbyDigital as well. Only the `audio_channels` are detected

**Comment by Toilal** [comment]
Main issue was fixed with #286 fix (https://github.com/guessit-io/guessit/commit/eb8229fd37c55a65e1f5502873a50b460149c27f), and I've added Dolby keyword as a DolbyDigital.
